app/templates/views/check/ok.html

```diff
@@ -19,12 +18,11 @@
 
 {% block maincolumn_content %}
 
-  {{ page_header('Preview of {}'.format(template.name)) }}
+  {{ page_header('Select delivery time') }}
 
   {{ template|string }}
   <div class="bottom-gutter-3-2">
-    <form method="post" enctype="multipart/form-data" action="{{url_for('main.start_job', service_id=current_service.id, upload_id=upload_id)}}" class='page-footer'>
-      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
```

Comment:
@ccostino I moved {{ csrf_token() }} to the new preview page, since this ok.html page for scheduling the message and not actually submitting until the preview page.

Reply:
We caught up about this and made a couple of adjustments, thanks again @heyitsmebev!